HTAN.model.csv

```diff
@@ -64,12 +64,40 @@ Imaging Level 1,Raw imaging data,,"Component, Filename, File Format, HTAN Parent
 SRRS Imaging Level 2,SRRS-specific HTAN raw and pre-processed image data,,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Data File ID, Channel Metadata Filename, Imaging Assay Type, Protocol Link, Software and Version, Microscope, Objective, NominalMagnification, Pyramid, Zstack, Tseries, Passed QC, Frame Averaging, Image ID, DimensionOrder, PhysicalSizeX, PhysicalSizeXUnit, PhysicalSizeY, PhysicalSizeYUnit, Pixels BigEndian, PlaneCount, SizeC, SizeT, SizeX, SizeY, SizeZ, PixelType",,FALSE,Assay,Biospecimen,,
 RPPA Level 2,Array based protemics. Each dilution curve of spot intensities is fitted using the monotone increasing B-spline model in the SuperCurve R package. This fits a single curve using all the samples on a slide with the signal intensity as the response variable and the dilution steps as independent variables. The fitted curve is plotted with the signal intensities on the y-axis and the log2-concentration of proteins on the x-axis for diagnostic purposes.,,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID, HTAN RPPA Antibody Table, Assay Type, Protocol Link, Software and Version",,FALSE,Assay,Biospecimen,,
 HTAN RPPA Antibody Table,A table containing antibody level metadata for RPPA,,"HTAN RPPA Antibody Table ID, Filename, File Format, Ab Name Reported on Dataset, GENCODE Gene Symbol Target, UNIPROT Protein ID Target, Phosphoprotein Flag, Vendor, Catalog Number, Internal Ab ID, Species, RPPA Dilution, Phospho Site, RPPA Validation Status, Clone, Clonality, Notes",,TRUE,RPPA Level 2,RPPA Level 2,,
-RPPA Level 3, "Level 3 Reverse Phase Protein Array (RPPA) data contains intra-batch normalized intensities.",,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID,  Assay Type, Software and Version, Normalization Method",,FALSE,Assay,Biospecimen,,
-RPPA Level 4, "description",,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, Assay Type, Batch Correction Method",,FALSE,Assay,Biospecimen,,
+RPPA Level 3," ""Level 3 Reverse Phase Protein Array (RPPA) data contains intra-batch normalized intensities.""",,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID,  Assay Type, Software and Version, Normalization Method",,FALSE,Assay,Biospecimen,,
+RPPA Level 4," ""description""",,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, Assay Type, Batch Correction Method",,FALSE,Assay,Biospecimen,,

```

Comment:
Looks like this quote change is due to not being level with `main`. Line should be unchanged.